instrumentation/opentelemetry-instrumentation-system-metrics/src/opentelemetry/instrumentation/system_metrics/__init__.py

```diff
@@ -16,8 +16,8 @@
 process (CPU, memory, garbage collection) metrics. By default, the
 following metrics are configured:
 
+"system_cpu_times": ["user", "system", "idle"],
```

Comment:
the labels here should be updated to reflect the new labels

Reply:
Right, fixing...

Reply:
Fixed